src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,10 @@
+import MuiAvatar from '@material-ui/core/Avatar';
+import React from 'react';
+
+export const Avatar = (props: any) => {
+
+	const source = props.image && props.image.uri ? props.image.uri : props.image;
```

Comment:
This will not work in all situations. For example if someone does this:

```jsx
import img from './path/to/image.jpg`

<Avatar type="image" image={img} />
```

This WILL fail!

Reply:
what should be the option for it kindly provide some code